ljpost/ljpost.php

```diff
@@ -162,11 +164,13 @@ function ljpost_send(&$a,&$b) {
 
 	$tz = 'UTC';
 
-	$x = q("select timezone from user where uid = %d limit 1",
+	$x = q("SELECT `timezone` FROM `user` WHERE `uid` = %d LIMIT 1",
 		intval($b['uid'])
 	);
-	if($x && strlen($x[0]['timezone']))
+
+	if (DBM::is_result($x) && strlen($x[0]['timezone'])) {
```

Comment:
Please use `!empty` instead of `strlen`.

Reply:
Yes, wrong usage of it.

Reply:
Strange, not auto-hiding this ...

Reply:
Because you didn't change it. It should be `if (!empty($x[0]['timezone'])) {`.

Reply:
And leave DBM::is_result() out?